NorthstarDedicatedTest/scriptsrson.cpp

```diff
@@ -11,6 +11,7 @@ void ModManager::BuildScriptsRson()
 	spdlog::info("Building custom scripts.rson");
 	fs::path MOD_SCRIPTS_RSON_PATH = fs::path(GetCompiledAssetsPath() / MOD_SCRIPTS_RSON_SUFFIX);
 	fs::remove(MOD_SCRIPTS_RSON_PATH);
+	std::vector<std::string> scriptsAdded;
```

Comment:
Could use a better variable name

Reply:
See resolved conversation below.
(This fix was removed.)